### PR TITLE
more compact form editor

### PR DESCRIPTION
**Pull request by opqdonut**
follow-up to #1804 #2597

related to #1899

# Checklist for author

Remove items that aren't applicable, check items that are done.

## Reviewability
- [x] Link to issue
- [ ] Consider adding screenshots for ease of review

## Documentation
- [ ] Update changelog if necessary
- [x] Components are added to guide page
- [ ] Update manual/ (if applicable)
  - will probably need some new screenshots for manual?

## Testing
- [ ] Valuable features are integration / browser / acceptance tested automatically

### CODE REVIEW

**Comment by opqdonut** [comment]
## Before

field-editor-before

## After

field-editor-after

File: src/cljs/rems/administration/components.cljs [hunk 1]
```diff
@@ -35,31 +36,39 @@
   [:div {:class "invalid-feedback"}
    (when error (text-format error label))])
 
-(defn input-field [{:keys [keys label placeholder context type normalizer readonly]}]
+(defn input-field [{:keys [keys label placeholder context type normalizer readonly inline?]}]
```
Comment (marharyta) [inline src/cljs/rems/administration/components.cljs:39]: not a fan of calling it inline, it is easily confused with a CSS property but do not have a strong opinion

Comment (opqdonut) [reply src/cljs/rems/administration/components.cljs:39]: We use `inline?` in rems.atoms/info-field and `inline-info-field` in rems.administration.components, that's where I copied it from. It's not ideal I agree.

Comment (Macroz) [reply src/cljs/rems/administration/components.cljs:39]: This is where it comes from:
https://getbootstrap.com/docs/4.0/components/forms/#inline

Comment (marharyta) [reply src/cljs/rems/administration/components.cljs:39]: I see, in any case, I think this is a great feature and ok to merge

File: src/cljs/rems/administration/create_form.cljs [hunk 2]
```diff
@@ -660,22 +664,23 @@
            [:div.form-field {:id (field-editor-id (:field/id field))
                              :key index
                              :data-field-index index}
-            [:div.form-field-header
-             [:h3 (text-format :t.create-form/field-n (inc index) (localized-field-title field @(rf/subscribe [:language])))]
-             [:div.form-field-controls
+            [:div.form-field-header.d-flex
+             [:h3
+              (text-format :t.create-form/field-n (inc index) (localized-field-title field @(rf/subscribe [:language])))]
+             [:div.form-field-controls.text-nowrap.ml-auto
```
Comment (Macroz) [inline src/cljs/rems/administration/create_form.cljs:670]: In the original I prefer the `h3` oneliner. For me it reads that ok here's the `h3` and I don't need to care about the rest. With the contents split to the next row I need to visually check that the one character indentation is there.

**Review by Macroz — APPROVED** [review]
Looks good, let's test the finished flow in the demo? I think we agreed on that already.

**Pull request merged**